tests/phpunit/includes/speed-trap-listener.php

```diff
@@ -1,10 +1,17 @@
 <?php
```

Comment:
The [phpunit-speedtrap](https://github.com/johnkary/phpunit-speedtrap/blob/master/composer.json#L20) supports PHP 7, 8, & 9, should we drop our custom implementation and load this via Composer now instead?

Reply:
No. The tests when run in Travis use the Travis shipped PHPUnit phar and don't do a Composer install, so the speedtrap would not be available then.

(This is part of the problem of why the tests weren't running on PHP 8, which is addressed and explained in detail in [Trac #50902](https://core.trac.wordpress.org/ticket/50902).